README.md

```diff
@@ -54,3 +54,6 @@ Sources
 
   * Django registration form template inspired from 
     https://github.com/macdhuibh/django-registration-templates/blob/master/registration/registration_form.html
+
+* Bootstrap  
+  http://getbootstrap.com/         
```

Comment:
You should cite the license of Bootstrap to remain consistent with attribution expectations. The Bootstrap framework is licensed under the MIT license.

https://github.com/twbs/bootstrap/blob/master/LICENSE

Quite possibly, add a new file named `Bootstrap.LICENSE` with the same contents as the license linked above.